servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpLoadBalancerFactory.java

```diff
@@ -230,5 +230,10 @@ public ReservedBlockingStreamingHttpConnection asBlockingStreamingConnection() {
         public ReservedBlockingHttpConnection asBlockingConnection() {
             return toReservedBlockingConnection(this, executionContext().executionStrategy());
         }
+
+        @Override
+        public String toString() {
+            return delegate.toString();
```

Comment:
we typically do something like `getClass().getSimpleName() + '(' delegate + ')'` does that make sense here (and other wrapper cases)?

Reply:
Then it's becomes a very long chain of delegation in RRLB logs. Let's keep it this way for now as these wrappers neither apply any user-visible behavior change for the connection nor add concurrency controller output in logs.